AccessibilitySnapshot/iOSSnapshotTestCase/Classes/FBSnapshotTestCase+Accessibility.swift

```diff
@@ -38,6 +39,7 @@ extension FBSnapshotTestCase {
     public func SnapshotVerifyAccessibility(
         _ view: UIView,
         identifier: String = "",
+        forceCursorVisible: Bool = true,
```

Comment:
I'm tempted to say this is a patch change since it addresses flaky tests, but I _think_ defaulting this to true makes it a breaking change, since tests can start failing if consumers already had a different fix in place - for example, pausing the animation in a similar way except setting the offset to a point where the cursor is hidden.

To be clear, making a breaking change is totally fine, just wanted to call it out here.

Reply:
Agreed, probably a breaking change.